src/store/mutations.js

```diff
@@ -0,0 +1,7 @@
+const mutations = {
+  SET_USER(state, data) {
+    state.user = data;
+  },
+};
+
```

Comment:
This is where we define our mutations, which are functions that allow use to change a state in the store (and thus throughout the app)

SET_USER gets the user state from /store/index.js but I'm not sure why it "= data"